packages/file_selector/file_selector_macos/macos/Classes/FileSelectorPlugin.swift

```diff
@@ -105,12 +105,12 @@ public class FileSelectorPlugin: NSObject, FlutterPlugin, FileSelectorApi {
         // that via the types; see messages.dart and https://github.com/flutter/flutter/issues/97848
         allowedTypes.append(contentsOf: acceptedTypes.utis.compactMap({ UTType($0!) }))
         allowedTypes.append(
-          contentsOf: acceptedTypes.extensions.flatMap({
-            UTType.types(tag: $0!, tagClass: UTTagClass.filenameExtension, conformingTo: nil)
```

Comment:
Just changing `nil` to `UTType.data` would fix the issue, but at that point it is (per Apple's docs) identical to the more readable `init` version I switched to.